### PR TITLE
closes 116 UI for hit validation process

### PR DESCRIPTION
**Pull request by Tremirre**
I implemented preliminary UI for the Hit Validation process. (No implementation for data processing, just the visual layer)

We have to discuss what else is missing from the UI (e.g. in terms of which params are to be set, what else should be visible in Hit Browser)

 # Screening Input

image

Currently only two params for concentration bounds can be set.

Data processing starts as soon as the user uploads a file.
Both parts have a message box to inform about interaction status:
image

# Hit Browser

image

Left panel allows to switch between different compounds data.
Middle panel shows IC50 graph and on the right there is a list of properties that are defined in the Hit Validation process (it might be extended, e.g. currently there is no comment field)

In the future we might add search box to the compounds list to allow to find a specific compound by ID.

# Report

image

Just two buttons without any callbacks.

---

Any suggestions/improvements are welcome 🐼

### CODE REVIEW

File: dashboard/pages/hit_validation/stages/s1_screening_data_input.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                        html.Label(
+                            children="Concentration lower bound (µM)",
+                            className="form-label",
+                        ),
+                        dcc.Input(
+                            id="concentration-lower-bound-input",
+                            type="number",
+                            value=0,
+                            min=0,
+                            step=0.0001,
```
Comment (azywot) [inline dashboard/pages/hit_validation/stages/s1_screening_data_input.py:21]: Should we define the minimum and the step?

**Review by azywot — APPROVED** [review]
Great job! 😎 This will simplify further steps a much 
Added just one minor comment

File: dashboard/pages/hit_validation/stages/s1_screening_data_input.py (hunk 1 above)
Comment (Tremirre) [reply dashboard/pages/hit_validation/stages/s1_screening_data_input.py:21]: step might not be necessary so I'll remove it,
but as for the minimum - I think that the concentration can't be lower than 0μM

**Review by zuzg — APPROVED** [review]
Nice! I would change the order to screening, correlation, hit
image

**Pull request merged**